File: ABC/351-400/376/a.py
def main():
  n,c = map(int,input().split()) 
  t = list(map(int,input().split()))
  last = -1e9
  ans = 0
  for i in range(n):
    if t[i]-last < c:
      continue
    last = t[i]
    ans+=1

  print(ans)

if __name__ == '__main__':
  main()

File: ABC/351-400/376/a.cpp
#include <bits/stdc++.h>
using namespace std;
#define rep(i,n) for(int i=0;i<n;++i)

int main(){
    int n,c;
    cin >> n >> c;
    int last = -1001001001;
    int ans = 0;
    for(int i=0;i<n;i++){
        int t;
        cin >> t;
        if(t-last<c) continue;
        last = t;
        ans++;
    }
    cout << ans << endl;
    return 0;
}